tests/format/js/conditional/jsfmt.spec.js

```diff
@@ -1 +1,3 @@
 run_spec(__dirname, ["babel", "flow", "typescript"]);
+run_spec(__dirname, ["babel", "flow", "typescript"], { printWidth: 18 });
+run_spec(__dirname, ["babel", "flow", "typescript"], { printWidth: 25 });
```

Comment:
Can you make the new tests longer to cross 80 width, so we can remove the `printWidth`?  Or you can put new tests into a new directory, the snap became very big, hard to review.